crd/nodenetworkconfig/api/v1alpha/nodenetworkconfig.go

```diff
@@ -65,6 +65,8 @@ type NCStatus string
 
 const (
 	NCStatusSubnetFull NCStatus = "SubnetFull"
```

Comment:
nit: Does `NCSubnetFull` aligns more with the `NCUpdateSuccess` and `NCUpdateFailed` ?

Reply:
So NCSubnetFull will be more accurate representation when the Subnet is Full error is present in DNC to pass on this error. NCUpdate Success is to overwrite when the NC was updated successfully from DNC and NCUpdateFailure is for any other error Updating the NC from DNC-RC's perspective. I am using this field to pass and communicate errors between DNC and the node services like CNS and CNI